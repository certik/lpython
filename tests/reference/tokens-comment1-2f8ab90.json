{
    "basename": "tokens-comment1-2f8ab90",
    "cmd": "lpython --no-color --show-tokens {infile} -o {outfile}",
    "infile": "tests/tokens/comment1.py",
    "infile_hash": "27258459bf39e8c5bcaa41e7fef8a593775a915273787edc46c73b4d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "tokens-comment1-2f8ab90.stdout",
    "stdout_hash": "1dd4691dd42cfd3ed3e6c7bfb783aa2b0977cf977bd68729f5264224",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}