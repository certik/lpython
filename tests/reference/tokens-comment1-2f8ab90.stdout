(TOKEN "eolcomment") 0:10
(KEYWORD "def") 11:13
(TOKEN "identifier" test) 15:18
(TOKEN "(") 19:19
(TOKEN ")") 20:20
(TOKEN ":") 21:21
(TOKEN "eolcomment") 23:34
(TOKEN "comment") 39:50
(TOKEN "indent") 51:54
(TOKEN "identifier" print) 55:59
(TOKEN "(") 60:60
(TOKEN ")") 61:61
(NEWLINE) 62:62
(TOKEN "dedent") 62:62
(TOKEN "comment") 63:74
(TOKEN "identifier" test) 75:78
(TOKEN "(") 79:79
(TOKEN ")") 80:80
(TOKEN "eolcomment") 82:93
(TOKEN "comment") 94:105
(NEWLINE) 106:106
(TOKEN "comment") 107:110
(TOKEN "comment") 111:123
(TOKEN "comment") 124:134
(EOF) 135:135
